scripts/db/db_migrate.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'mysql2'
+
+HOST = 'localhost'
+PORT = '3306'
+USER = 'test_bot'
+PASSWORD = 'pass'
```

Comment:
Read the username and password from the same file as write_build_results.rb file or pass them via CLI parameters.